docs/runtime.md

```diff
@@ -0,0 +1,72 @@
+# On the runtime of feature attribution methods
+
+Feature attribution methods don't only strongly differ in the output of the most important features, but also in run time.
+Hence, the selection of the feature attribution method of choice should depend on the model at hand and any potential limitations of computational resources.
+
+In the following sections we will briefly discuss the complexity of the respective feature attribution methods and provide further intuition by describing empirical run time results of common models.
+
+## Feature ablation
+
+The general approach of feature ablation can be summarized as follows:
+
+1. Train the model on a train set and calculate a score on the test set using any metric.
+2. For each of the m features, remove it from the training data and train the model. Then, calculate the score on the test set.
+3. Rank the features by the difference between the original score (from the model with all features) and the score for the model using all features but one.
+
+Reference: https://www.samueltaylor.org/articles/feature-importance-for-any-model.html
+
+The complexity of feature ablation is henceforth: O(n * m²) where n == number of observations and m == number of features
```

Comment:
Can we move `n`, `m`, and other variable definitions to the top? And then have anything that's explainer-specific written out in the O notation.

Also, I've always wondered this but why do you use == instead of = ???

Reply:
Oh, his notation is pure hell. If I do it wrong, I get reminded that I need to fix it :)

Just wanted to get feedback on the idea. We can also render proper math in Sphinx to make this prettier.